ci-operator/jobs/kubevirt/hyperconverged-cluster-operator/kubevirt-hyperconverged-cluster-operator-master-presubmits.yaml

```diff
@@ -262,6 +262,7 @@ presubmits:
       pj-rehearse.openshift.io/can-be-rehearsed: "true"
     name: pull-ci-kubevirt-hyperconverged-cluster-operator-master-hco-e2e-upgrade-aws
     rerun_command: /test hco-e2e-upgrade-aws
+    run_if_changed: ^(pkg/*|deploy/crds/*|deploy/index-image/*|hack/*|build/*|Makefile|go\.mod|go\.sum)$
```

Comment:
I'd say for sure also `cmd`, maybe even `vendor` (but `go.sum` should already detect a change...).